OpenIndiana uses the folder vendor-packages instead of site-packages.

diff -ur bind-9.14.3/bin/python/dnssec-checkds.py.in bind-9.14.3-patched/bin/python/dnssec-checkds.py.in
--- bind-9.14.3/bin/python/dnssec-checkds.py.in	2019-06-04 18:20:04.000000000 +0000
+++ bind-9.14.3-patched/bin/python/dnssec-checkds.py.in	2019-06-23 16:46:42.639989194 +0000
@@ -19,7 +19,7 @@
         sys.path.insert(1, '@PYTHON_INSTALL_DIR@')
     else:
         sys.path.insert(1, os.path.join('@prefix@', 'lib',
-                        'python' + sys.version[:3], 'site-packages'))
+                        'python' + sys.version[:3], 'vendor-packages'))
 
 import isc.checkds
 
diff -ur bind-9.14.3/bin/python/dnssec-coverage.py.in bind-9.14.3-patched/bin/python/dnssec-coverage.py.in
--- bind-9.14.3/bin/python/dnssec-coverage.py.in	2019-06-04 18:20:04.000000000 +0000
+++ bind-9.14.3-patched/bin/python/dnssec-coverage.py.in	2019-06-23 16:46:27.245745285 +0000
@@ -19,7 +19,7 @@
         sys.path.insert(1, '@PYTHON_INSTALL_DIR@')
     else:
         sys.path.insert(1, os.path.join('@prefix@', 'lib',
-                        'python' + sys.version[:3], 'site-packages'))
+                        'python' + sys.version[:3], 'vendor-packages'))
 
 import isc.coverage
 
diff -ur bind-9.14.3/bin/python/dnssec-keymgr.py.in bind-9.14.3-patched/bin/python/dnssec-keymgr.py.in
--- bind-9.14.3/bin/python/dnssec-keymgr.py.in	2019-06-04 18:20:04.000000000 +0000
+++ bind-9.14.3-patched/bin/python/dnssec-keymgr.py.in	2019-06-23 16:46:57.198420733 +0000
@@ -19,7 +19,7 @@
         sys.path.insert(1, '@PYTHON_INSTALL_DIR@')
     else:
         sys.path.insert(1, os.path.join('@prefix@', 'lib',
-                        'python' + sys.version[:3], 'site-packages'))
+                        'python' + sys.version[:3], 'vendor-packages'))
 
 import isc.keymgr
 
